### ISSUE TITLE
home directory filepath isn't parsed correctly when run interactively

### ISSUE THREAD

**Issue by zgulde** [zgulde/tomcat-setup#2]
Discovered when running a migration.

If prompted for the filepath interactively, and the filepath includes a `~`, the script fails and cannot find the file.

**Comment by zgulde** [comment]
This does it...

```bash
if grep '^~' <<< "$test"; then
    test=$(perl -pe "s!~!$HOME!" <<< $test)
fi
```

**Comment by zgulde** [comment]
fixed in fb2180b
